torat_server/server.go

```diff
@@ -117,6 +135,13 @@ func accept(conn net.Conn) {
 
 	saveData()
 
+	// Remove previous dead entry if it exists
+	for i, c := range activeClients {
+		if c.Hostname == ac.Hostname {
+			activeClients = append(activeClients[:i], activeClients[i+1:]...)
+		}
+	}
```

Comment:
I think this might introduce a bug where for whatever reason 2 client instances run on the same machine, this should therefore also check if the old session still works and keep it in that case.